### PR TITLE
feat: set credential and project setting for xsync

### PR DESCRIPTION
**Pull request by rkakodkar**
Add Xsync project settings and credentials.

### CODE REVIEW

File: xnat_admin_tools/create_new_projects.py [hunk 1]
```diff
@@ -50,6 +50,96 @@ def remove_empty(values_to_select, project_values):
...
+        "subject_assessors": {"sync_type": "none"},
+        "imaging_sessions": {"sync_type": "all"},
+        "enabled": True,
+        "source_project_id": project_id,
+        "sync_frequency": "on demand",
+        "sync_new_only": True,
+        "identifiers": "use_local",
+        "remote_url": xserver_host,
+        "remote_project_id": project_id,
+        "notification_emails": "",
```
Comment (mirestrepo) [inline xnat_admin_tools/create_new_projects.py:125]: I wonder if we should try to notify the users?

**Review by mirestrepo — COMMENTED** [review]
I'm going to wait on approving this as I have some questions about the username and password. I noticed that in the database, things look differently when done via UI, or vs this. I have a slack message scheduled for you and Brad with more details for tomorrow AM, we can chat more soon

File: xnat_admin_tools/create_new_projects.py (hunk 1 above)
Comment (rkakodkar) [reply xnat_admin_tools/create_new_projects.py:144]: I guess we're keeping this on hold based on the conversation in last BNC meeting

**Comment by rkakodkar** [comment]
> I'm going to wait on approving this as I have some questions about the username and password. I noticed that in the database, things look differently when done via UI, or vs this. I have a slack message scheduled for you and Brad with more details for tomorrow AM, we can chat more soon

Changes done. Using tokens in the latest commit.

**Review by mirestrepo — APPROVED** [review]
Looks great! 🚀

**Pull request merged**